### PR TITLE
CLDR-13890 world

### PR DESCRIPTION
**Pull request by Kherishma**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13890
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by srl295** [comment]
@Kherishma here is the issue from the run https://github.com/unicode-org/cldr/pull/916/checks?check_run_id=1696715159

```
    TestVariantName {
265
      Error: (TestCLDRUtils.java:60) : Test variant formatting combinedLong eo_001: expected "Esperanto (World)", got "Esperanto (world)"
266
      Error: (TestCLDRUtils.java:60) : Test variant formatting combinedShort eo_001: expected "Esperanto (World)", got "Esperanto (world)"
267
      Error: (TestCLDRUtils.java:60) : Test variant formatting uncombinedLong eo_001: expected "Esperanto (World)", got "Esperanto (world)"
268
      Error: (TestCLDRUtils.java:60) : Test variant formatting uncombinedShort eo_001: expected "Esperanto (World)", got "Esperanto (world)"
269
    } (0.037s) FAILED (4 failure(s))
270
```

The file `cldr/tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRUtils.java` has a line

```java
        checkNames(english, "eo_001", "Esperanto (World)");
```

can you correct it also?

File: common/subdivisions/en.xml [hunk 1]
```diff
@@ -5574,7 +5574,7 @@ Generated by tool:	SubdivisionNode
 			<subdivision type="zafs">Free State</subdivision>
 			<subdivision type="zagp">Gauteng</subdivision>
 			<subdivision type="zagt">Gauteng</subdivision>
-			<subdivision type="zakzn">Kwazulu-Natal</subdivision>
+			<subdivision type="zakzn">KwaZulu-Natal</subdivision>
```
Comment (srl295) [inline common/subdivisions/en.xml:5577]: isn't this part of #913 ?

**Comment by srl295** [comment]
@Kherishma https://github.com/Kherishma/cldr/blob/f7839a9bf2787c955badba8e7322e24480634ac3/tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCLDRUtils.java#L60

I got there by clicking on [Kherishma:CLDR-13890-World](https://github.com/Kherishma/cldr/tree/CLDR-13890-World) in the header and then "go to file…" and typing `TestCLDRUtils.java`

File: common/subdivisions/en.xml (hunk 1 above)
Comment (Kherishma) [reply common/subdivisions/en.xml:5577]: I don't believe so. I have already squashed and merged the PR for Kwazulu-Natal

**Pull request closed without merging**